rslib/Cargo.toml

```diff
@@ -109,6 +109,7 @@ utime = "0.3.1"
 workspace-hack = { version = "0.1", path = "../tools/workspace-hack" }
 zip = { version = "0.6.3", default-features = false, features = ["deflate", "time"] }
 zstd = { version = "0.12.2", features = ["zstdmt"] }
+palette = "0.6.1"
```

Comment:
Not currently used?

Reply:
The plan is to use it to generate colors that can be derived, e.g. `BUTTON_GRADIENT_START` and `...-END` from the base color `BUTTON` by making it slightly lighter/darker. The crate seems like a good fit for that purpose.

Generating helper colors from a set of base colors allows us to keep the amount of colors users need to tweak for new themes low, while also ensuring basics like hover feedback work at all times.